projects/storefrontlib/cms-components/myaccount/my-coupons/coupon-card/coupon-card.component.html

```diff
@@ -25,11 +25,19 @@
           </div>
         </div>
       </div>
-
-      <a (click)="readMore()" class="cx-card-read-more">{{
-        'myCoupons.readMore' | cxTranslate
-      }}</a>
-
+      <a
```

Comment:
Please add TODO comment how to deprecate this.

Reply:
And update breaking changes document `6_0.md`.

Reply:
done thanks